### PR TITLE
Feature/93 hook up algolia with businesses search

### PR DESCRIPTION
**Pull request by llexical**
# Describe your PR

Related to #
Fixes #

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1.

### CODE REVIEW

**Comment by jbolda** [comment]
I thought it worthwhile to test out an alternative approach then what you have here @llexical. This was what I was trying to describe in our DMs, but was struggling to adequately explain. So as you mentioned, code speaks 😄 => https://github.com/jbolda/RBB-Website/pull/1 (I opened a PR within my repo to provide a diff, but not confuse others by opening against this main repo.)

**Comment by magnificode** [comment]
Hey @llexical - apologize for polluting the history here 🤦‍♂️ - tried a few things that ended up stepping on your toes a bit too much. Totally me and my own exhaustion at fault here for not splitting off on a different branch.

I did make one change and moved the Application ID and the Search-Only API Key to a `.env.development` file. the `.env.development` file will need to be created on your local in addition to the `.env` file for Gatsby to properly grab on the client side.

Additionally, merging master into this branch triggered the auto linting on several other files 🤦‍♂️. That's where all the rest of those additions came from.

Long story short I just jumped in here, ran around the place waving my arms and knocking things over.

Because the API key was within the JS here I did regenerate the Algolia API's. But once they're dropped into that environment everything should be working as it did before I got in here and messed up the joint!

Apologies again. I'm going to get some sleep :)
